tests/main.rs

```diff
@@ -110,15 +135,92 @@ fn it_connects() {
         |_, _| unreachable!(),
         |_| unreachable!(),
         |_, _, _| unreachable!(),
+        |_, _| unreachable!(),
     ).test(|_| {})
 }
 
+// The current version of the mysql client does not support the select_db()
+// function. A pull request has been accepted to that project
+// (https://github.com/blackbeam/rust-mysql-simple/pull/171) however it
```

Comment:
Just make the `mysql` dependency a git one for the time being. By the time I make another release, I'm sure there'll have been another `mysql` release :)